src/js/helpers/fetchData.js

```diff
@@ -1,4 +1,13 @@
 import axios from 'axios';
+import { GraphQLClient } from 'graphql-request';
+
+export const createGraphQLClientsByLang = lang => {
```

Comment:
~nice! I'm thinking whether this function would save us any code in the static.config.js file~

Reply:
derp, nvm. you did it already.